sdk/search/search-documents/src/searchIndexClient.ts

```diff
@@ -812,8 +812,20 @@ export class SearchIndexClient {
    * Retrieves the SearchClient corresponding to this SearchIndexClient
    * @param indexName - Name of the index
    * @param options - SearchClient Options
+   * @typeParam Model - An optional TS interface that matches the type of the
```

Comment:
```suggestion
   * @typeParam Model - An optional type that represents the
```

No need to say it's a TS interface.

Reply:
Fixed!